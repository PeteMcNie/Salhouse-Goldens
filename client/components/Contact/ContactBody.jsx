import React from 'react'

import Image from 'react-bootstrap/Image'

function ContactBody () {
  return (
    <>
      <div className='belowNavbar'>
        <h2>Contact Us</h2>
        <p>Phone: +64 4 566 0561</p>
        <p>Email: <a href='mailto:cmcnie@xtra.co.nz'>cmcnie@xtra.co.nz</a></p>
        <p>Lower Hutt, New Zealand</p>
      </div>
      <div>
        <Image src='images/contact1.jpg' fluid />
      </div>
    </>
  )
}

export default ContactBody
