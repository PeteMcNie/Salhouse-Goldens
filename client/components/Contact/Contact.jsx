import React from 'react'

import Navbar from '../NavbarComp'
import ContactBody from './ContactBody'

function Contact () {
  return (
    <>
      <Navbar />
      <ContactBody />
    </>
  )
}

export default Contact
