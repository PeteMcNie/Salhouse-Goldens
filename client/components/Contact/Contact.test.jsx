import React from 'react'
import { render } from '@testing-library/react'

import Contact from './Contact'

test('Contact component to match snapshot', () => {
  const { container } = render(<Contact />)
  expect(container).toMatchSnapshot()
})
