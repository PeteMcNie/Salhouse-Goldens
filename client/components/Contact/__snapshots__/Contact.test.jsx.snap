// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact component to match snapshot 1`] = `
<div>
  <nav
    class="navbar navbar-expand-lg navbar-light bg-light fixed-top"
  >
    <a
      class="navbar-brand"
      href="/"
    >
      <img
        alt="Goldens retrievers, phoenix and kula"
        class="shlogo d-inline-block align-top"
        src="images/logo1.png"
      />
    </a>
    <button
      aria-controls="responsive-navbar-nav"
      aria-label="Toggle navigation"
      class="navbar-toggler collapsed"
      type="button"
    >
      <span
        class="navbar-toggler-icon"
      />
    </button>
    <div
      class="navbar-collapse collapse"
      id="responsive-navbar-nav"
    >
      <div
        class="ml-auto navbar-nav"
      >
        <a
          class="nav-link"
          data-rb-event-key="/"
          href="/"
        >
          Home
        </a>
        <div
          class="dropdown nav-item"
        >
          <a
            aria-expanded="false"
            aria-haspopup="true"
            class="dropdown-toggle nav-link"
            href="#"
            id="collasible-nav-dropdown"
            role="button"
          >
            Our Goldens
          </a>
        </div>
        <div
          class="dropdown nav-item"
        >
          <a
            aria-expanded="false"
            aria-haspopup="true"
            class="dropdown-toggle nav-link"
            href="#"
            id="collasible-nav-dropdown"
            role="button"
          >
            Grooming Your Golden
          </a>
        </div>
        <a
          class="nav-link"
          data-rb-event-key="/mylifewithanimals"
          href="/mylifewithanimals"
        >
          My Life With Animals
        </a>
        <div
          class="dropdown nav-item"
        >
          <a
            aria-expanded="false"
            aria-haspopup="true"
            class="dropdown-toggle nav-link"
            href="#"
            id="collasible-nav-dropdown"
            role="button"
          >
            About Us
          </a>
        </div>
        <a
          class="nav-link"
          data-rb-event-key="/links"
          href="/links"
        >
          Links
        </a>
        <a
          class="nav-link"
          data-rb-event-key="/contact"
          href="/contact"
        >
          Contact
        </a>
      </div>
    </div>
  </nav>
  <div
    class="mainBodyMLWA"
  >
    <div
      class="container"
    >
      <div
        class="row row"
      >
        <div
          class="contact col-lg-6"
        >
          <h2
            class="bodyTitle"
          >
            Contact
          </h2>
          <div
            class="contactInfo"
          >
            <p
              class="contactText"
            >
              Phone: +64 4 566 0561 or 0274164687
            </p>
            <p
              class="contactText"
            >
              Email: 
              <a
                href="mailto:cmcnie@xtra.co.nz"
              >
                cmcnie@xtra.co.nz
              </a>
            </p>
            <p
              class="linkslastline contactText"
            >
              Lower Hutt, New Zealand
            </p>
          </div>
        </div>
        <div
          class="col-lg-6"
        >
          <div
            class="imagelastline"
          >
            <img
              class="linksimage img-fluid rounded-circle"
              src="images/contact1.jpg"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
